api/data_refinery_api/views/experiment_document.py

```diff
@@ -281,7 +282,7 @@ class ExperimentDocumentView(DocumentViewSet):
         "accession_code": "accession_code",
         "alternate_accession_code": "alternate_accession_code",
         "platform": "platform_accession_codes",
-        "organism": "organism_names.raw",
+        "organism": "downloadable_organism_names.raw",
```

Comment:
This change looks like it will force everyone to search by downloadable_organism_names, just because that's the behavior we want in the frontend.

Could we instead a `downloadable_organism` filter field and then have the frontend use that? Or would the frontend then just show both filters and confuse users?